lsp-semantic-tokens.el

```diff
@@ -44,6 +44,23 @@ or modifier type returned by a language server has no face associated with it."
   :group 'lsp-semantic-tokens
   :type 'boolean)
 
+(defcustom lsp-semantic-tokens-allow-ranged-requests t
+  "Whether to use ranged semantic token requests when available.
+
+Note that even when this is set to t, delta requests will
+be preferred whenever possible, unless
+`lsp-semantic-tokens-allow-delta-requests' is false."
+  :group 'lsp-mode
```

Comment:
Could you change the groups to lsp-semantic-tokens to make those variables documentation show in correct sections of webpage?